projects/libraw/libraw_fuzzer.cc

```diff
@@ -20,6 +20,16 @@ limitations under the License.
 
 #include <libraw.h>
 
+enum InterpolationOptions {
```

Comment:
Import it from code rather then redefining here (if possible).

Reply:
Unfortunately the actual numbers are hard coded in the code. See https://github.com/LibRaw/LibRaw/blob/4a39decd113ca1c08db5b5ed1c9e96f6867e5f0e/src/postprocessing/dcraw_process.cpp#L156

Reply:
fine as-is then.